habitat-lab/habitat/config/default.py

```diff
@@ -57,6 +73,19 @@ def get_config(
             overrides=overrides if overrides is not None else [],
         )
 
+    # In the single-agent setup use the agent's key from `habitat.simulator.agents`.
+    sim_config = cfg.habitat.simulator
+    if len(sim_config.agents) == 1:
```

Comment:
Let's remove this non documented functionality. We plan to create "main_agent" for this purpose to be the default, those it won't  be needed..

Reply:
We agreed to create just an alias. Actually, I'd leave it as it is forcing users to explicitly define / chose the agent. @vincentpierre your thoughts?

Reply:
I think we should leave it as is. I do not want to have `main_agent` be defined in code (and that includes the default_structured_config file) but only be defined in the yamls that we provide. Nothing should prevent the user to name their unique agent anything they want and not need to modify agent_order

Reply:
I don't like the term "main" agent as "main" is an very overloaded term. Is there a better term we can use? Even agent_0 is more specific and easier to understand in my opinion.